### ISSUE TITLE
Flaky test in ConnectionPoolSpec

### ISSUE THREAD

**Issue by agolubev** [akka/akka#19083]
https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/4365/consoleFull
```
should not open a second connection if there is an idle one available *** FAILED *** (3 seconds, 96 milliseconds)
[info]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsgClass waiting for class akka.stream.testkit.TestSubscriber$OnNext
[info]   at scala.Predef$.assert(Predef.scala:179)
[info]   at akka.testkit.TestKitBase$class.expectMsgClass_internal(TestKit.scala:423)
[info]   at akka.testkit.TestKitBase$class.expectMsgType(TestKit.scala:396)
[info]   at akka.testkit.TestKit.expectMsgType(TestKit.scala:718)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectNext(StreamTestKit.scala:254)
[info]   at akka.http.impl.engine.client.ConnectionPoolSpec$$anonfun$7$$anonfun$apply$mcV$sp$4$$anon$7.<init>(ConnectionPoolSpec.scala:126)
[info]   at akka.http.impl.engine.client.ConnectionPoolSpec$$anonfun$7$$anonfun$apply$mcV$sp$4.apply$mcV$sp(ConnectionPoolSpec.scala:115)
[info]   at akka.http.impl.engine.client.ConnectionPoolSpec$$anonfun$7$$anonfun$apply$mcV$sp$4.apply(ConnectionPoolSpec.scala:115)
[info]   at akka.http.impl.engine.client.ConnectionPoolSpec$$anonfun$7$$anonfun$apply$mcV$sp$4.apply(ConnectionPoolSpec.scala:115)
```

**Comment by ktoso** [comment]
thanks for reporting

**Comment by johanandren** [comment]
Again, but actually not in RenderingSpec, it's ConnectionPoolSpec.

https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/5028/consoleFull

**Comment by patriknw** [comment]
@johanandren do you know if we have fixed things related to connection pool that makes this test failure obsolete?

**Comment by johanandren** [comment]
Could perhaps have been affected/fixed by 81579bd

**Comment by patriknw** [comment]
then, let's hope that is it ;-)
